src/plugins/stringStyle.js

```diff
@@ -0,0 +1,32 @@
+/* @flow */
+import createRule from '../utils/createRule'
+import type {JssStyle, RuleOptions, Rule} from '../types'
+
+const nlOrSemi = /\n|;/
```

Comment:
Hmm maybe for such a simplistic parse its better to enforce newlines and semicolons? Meaning I would split by semicolon + newline and not support one liners and no semicolons.